core/crdt/composite.go

```diff
@@ -92,39 +92,19 @@ func (delta *CompositeDAGDelta) Links() []core.DAGLink {
 
 // CompositeDAG is a CRDT structure that is used to track a collection of sub MerkleCRDTs.
 type CompositeDAG struct {
-	store datastore.DSReaderWriter
-	key   core.DataStoreKey
-	// schemaVersionKey is the schema version datastore key at the time of commit.
-	//
-	// It can be used to identify the collection datastructure state at time of commit.
-	schemaVersionKey core.CollectionSchemaVersionKey
-
-	fieldName string
+	baseCRDT
 }
 
-var _ core.ReplicatedData = CompositeDAG{}
```

Comment:
question: Why has this been removed?

Reply:
Because it's already covered at the top of the file :)

Reply:
✅